package.json

```diff
@@ -36,18 +36,15 @@
     "test": "gulp test"
   },
   "dependencies": {
+    "carbonldp-ts-docs-engine": "file:../../../carbonldp-ts-docs-engine/carbonldp-ts-docs-engine-0.1.0.tgz",
```

Comment:
Remove this dependency.
When published the `carbonldp-ts-docs-engine` package, this should be added as a dev-dependency